### PR TITLE
Fix type that was blocking the build process

### PR DESCRIPTION
**Pull request by WiXSL**
[Trello Card #801](https://trello.com/c/Nh9y7LJs/801-ra-enterprise-demo-project-build-process-is-failing) 

- [x] Completed

## Release process

- [X] Select a Github label (**WIP** or **RFR**)

### CODE REVIEW

File: src/dashboard/OrderChart.tsx [hunk 1]
```diff
@@ -104,8 +104,8 @@ const OrderChart = (props: { orders?: Order[] }) => {
                             <CartesianGrid strokeDasharray="3 3" />
                             <Tooltip
                                 cursor={{ strokeDasharray: '3 3' }}
-                                formatter={(value: number) =>
-                                    formatNumberAsUSD(value, 2)
+                                formatter={(value: any) =>
```
Comment (WiXSL) [inline src/dashboard/OrderChart.tsx:107]: Probably not the best fix, but I could get why typescript was complaining

File: src/dashboard/OrderChart.tsx [hunk 2]
```diff
@@ -104,8 +104,8 @@ const OrderChart = (props: { orders?: Order[] }) => {
...
+                                    formatNumberAsUSD(Number(value), 2)
```
Comment (WiXSL) [inline src/dashboard/OrderChart.tsx:108]: Probably not the best fix, but I could get why typescript was complaining plus this allowed me to build locally

Comment (WiXSL) [reply src/dashboard/OrderChart.tsx:108]: Maybe I can simply @ts-ignor it

**Comment by slax57** [comment]
Superseded by https://github.com/react-admin/ra-enterprise/pull/636

**Pull request closed without merging**